src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -339,11 +350,13 @@ public override void VisitMethod(IMethodSymbol symbol)
             {
                 case MethodKind.Ordinary:
                 case MethodKind.DelegateInvoke:
-                case MethodKind.ReducedExtension:
                 case MethodKind.LocalFunction:
                     //containing type will be the delegate type, name will be Invoke
                     builder.Add(CreatePart(SymbolDisplayPartKind.MethodName, symbol, symbol.Name));
                     break;
+                case MethodKind.ReducedExtension:
+                    builder.Add(CreatePart(SymbolDisplayPartKind.ExtensionMethodName, symbol, symbol.Name));
+                    break;
```

Comment:
what about extension methods that are not reduced?

Reply:
They will be tagged as methods in the same way that the `NameSyntaxClassifier` classifies them.  I will add a comment.

Reply:
I guess my question is more: is this the behavior we want?  Should we always be classifying extension methods in the same way regardless of how they are invoked?  I'm open either way.